packages/flutter/lib/src/material/text_field.dart

```diff
@@ -302,6 +310,27 @@ class TextField extends StatefulWidget {
   /// {@macro flutter.widgets.editableText.readOnly}
   final bool readOnly;
 
+  /// Whether to show cut option in context menu
```

Comment:
why not use macros for these as well?

Reply:
It has a different default from editable text. same for copy